server/utils/passport/ensure-user.js

```diff
@@ -0,0 +1,35 @@
+import { routes } from '../routes'
+import { ADMIN_ROLE } from '../constants'
+
+export default async function ensurePermission(req, res, next) {
```

Comment:
```suggestion
export default async function ensureUser(req, res, next) {
```

Reply:
There's already an ensureAdmin middleware, but I agree this should be renamed, what do you think about ensureActiveUser

Reply:
I think we need to start with what this is doing. Is this ensuring the user is active or that the user is an admin? What is the difference between this and the other `ensureAdmin` middleware?

Reply:
the ensureAdmin middleware is strictly checking for the admin role, this ensureUser is checking that the user is blocked, has access to view site data

Reply:
gotcha, that's helpful. Should we keep the existing `ensurePermission` then? That actually seems pretty well named.

Reply:
Yes I think we can keep it and just remove the role part

Reply:
https://github.com/AMP-SCZ/dpdash/pull/237/commits/991410c2b7d76f49852b5c3ebe79dfa0bcb18e36